torchvision/ops/poolers.py

```diff
@@ -35,7 +51,7 @@ def __call__(self, boxlists):
         s = torch.sqrt(torch.cat([box_area(boxlist) for boxlist in boxlists]))
 
         # Eqn.(1) in FPN paper
-        target_lvls = torch.floor(self.lvl0 + torch.log2(s / self.s0 + self.eps))
+        target_lvls = torch.floor(self.lvl0 + torch.log2(s / self.s0) + torch.tensor(self.eps, dtype=torch.float32))
```

Comment:
can we use instead the type of `s` here? As we could potentially have a `s` be a double (or half) tensor, and this would fail (prior to the type promotion PR, which has been recently merged)